applications/application_template.md

```diff
@@ -1,128 +0,0 @@
-# Crust Grant Proposal
```

Comment:
You can't remove or change the original file, this is described in [Application Process](https://github.com/crustio/Crust-Grants-Program#1-application) clearly right?